addons/website_event/views/event_templates_list.xml

```diff
@@ -88,26 +88,29 @@
                         <t t-foreach="categories" t-as="category">
                             <div class="accordion-item">
                                 <h2 class="accordion-header">
-                                    <button class="accordion-button border-top collapsed"
+                                    <button class="accordion-button collapsed"
                                         type="button"
                                         data-bs-toggle="collapse"
                                         t-att-data-bs-target="'.o_wevent_offcanvas_cat_%s' % category.name"
-                                        aria-expanded="false">
+                                        aria-expanded="false"
+                                        t-attf-aria-controls="o_wevent_offcanvas_cat_#{category.name}">
```

Comment:
Seems like this makes more sense above `t-att-data-bs-target`, also this doesn't seem to be mentioned in the commit